packages/core/src/js/sdk.tsx

```diff
@@ -157,6 +157,7 @@ export function wrap<P extends Record<string, unknown>>(
   const profilerProps = {
     ...(options?.profilerProps ?? {}),
     name: RootComponent.displayName ?? 'Root',
+    updateProps: {}
```

Comment:
Is this required? I don't know the prop.

Reply:
It was always required, but for for some reason it only started to complain once I started working on V9
https://github.com/getsentry/sentry-javascript/blob/6914a2453472b3683fe76e67e248414d7ec4ec06/packages/react/src/profiler.tsx#L24

Reply:
Okay, I see, thank you for the details.

Could you open a follow-up PR with `options?.profilerProps` type update to exclude the update props and children. https://github.com/getsentry/sentry-javascript/blob/1048a437b09955d31118960624b6d58242389c45/packages/react/src/profiler.tsx#L158